doc-content/kogito-docs/src/main/asciidoc/openshift/con_kogito-on-ocp.adoc

```diff
@@ -0,0 +1,10 @@
+[id='con_kogito-on-ocp']
+
+= {PRODUCT} on {OPENSHIFT}
+
+You can deploy {PRODUCT} services on {OPENSHIFT} for cloud implementation. In this solution, {PRODUCT} services are deployed as OpenShift pods that you can scale up and down individually to provide as few or as many containers as required for a particular service. You can use standard OpenShift methods to manage the pods and balance the load.
```

Comment:
shouldn't   In this solution be " In this archteture"?

Reply:
@spolt, sure.